### DISCUSSION TITLE
Issue in Creating Iceberg table using Trino

### DISCUSSION THREAD

**Discussion by empenapps** [trinodb/trino#23724] [Q&A]
I am doing a POC.
I have setup minio, iceberg-rest and trino using docker compose.
I have configured iceberg.properties to use rest-catalog.

when I am trying to create a table using trino, it says no factory for location.
when i check logs of iceberg-rest, it says no such table found exception.

**Comment by realknorke** [comment]
do you have the native reader enabled?

**Comment by Lords08** [comment]
I also experienced it.

SHOW SCHEMA; success.
SHOW TABLES; success.
But when running the query SELECT * FROM catalog.schema.table;

an error appears:
Query 20241217_065327_00002_qau87 failed: No factory for location xxxxxx.

I use docker compose for deployment, the images I use is like this:
1. quay.io/minio/minio:latest
2. tabulario/iceberg-rest
3. tabulario/spark-iceberg
4. minio/mc:latest
5. trinodb/trino:latest

---------------------------
EDIT:

Now I can create table, show all tables, and select * from table in trino using the configuration in this link:
https://github.com/trinodb/trino/issues/23740#issuecomment-2513552874

My iceberg.properties:
```
connector.name=iceberg
iceberg.catalog.type=rest
iceberg.rest-catalog.uri=http://rest:8181
iceberg.rest-catalog.warehouse=s3a://warehouse/
iceberg.file-format=PARQUET
fs.native-s3.enabled=true
s3.endpoint=http://minio:9000
s3.region=us-east-1
s3.path-style-access=true
s3.aws-access-key=xxxx
s3.aws-secret-key=xxxx
iceberg.compression-codec=ZSTD
```
